docs/data-sources/activity_definitions.md

```diff
@@ -132,6 +133,27 @@ Read-Only:
 - `unit_id` (String)
 
 
+<a id="nestedobjatt--content--argument_definitions--attributes--fields"></a>
```

Comment:
I don't know what I'm doing  :)

Reply:
I suppose it was auto-generated, right?

Reply:
ah I think you're right, we can generate this - forgot about it, thanks - will remove my manual changes then and generate it

Reply:
actually the doc is generated by an automated workflow, right?

Reply:
It looks like it's done [here ](https://github.com/leanspace/terraform-provider-leanspace/blob/b5592ee3a408ccd77bacfb2a8697edf6920c1aee/.github/workflows/release.yml#L106)during the release